Profile/__test__/handler.test.ts

```diff
@@ -0,0 +1,77 @@
+import * as E from "fp-ts/lib/Either";
+import { FiscalCode } from "../../generated/definitions/fast-login/FiscalCode";
+import { SpidLevel } from "../../utils/enums/SpidLevels";
+import { IHslJwtPayloadExtended } from "../../utils/middlewares/hsl-jwt-validation-middleware";
+import { profileHandler } from "../handler";
+import { Client } from "../../generated/definitions/io-functions-app/client";
+
+// #region mocks
+const getProfileMock = jest.fn(async () =>
+  E.right({
+    status: 200,
+    value: {
+      email: "example@test.com"
+    }
+  })
+);
+const functionsAppClientMock = ({
+  getProfile: getProfileMock
+} as unknown) as Client<"SubscriptionKey">;
+
+const getProfileNotFoundMock = jest.fn(async () =>
```

Comment:
may I suggest to remove this mock and use the already defined `getProfileMock`? you can use `mockReturnValueOnce` function in each test later to edit the default behaviour you described in line 9